src/WebUI/ClientApp/src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+import { Router } from "@angular/router";
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+})
+export class DashboardComponent {
+
+    public _router;
+
+    constructor(private router:Router){
+        this._router = router;
+    }
+
+    CreatePPTClick(){
```

Comment:
❗ javascript world uses `camelCase` for code styling

Reply:
Done